### ISSUE TITLE
Temp fix for   Warning: React depends on requestAnimationFrame.

### ISSUE THREAD

**Issue by QuantumInformation** [wmonk/create-react-app-typescript#212]
Does anyone have a way to get rid of this error when running tests? I know that there is some downstream fixes but I don't want to wait

![image](https://user-images.githubusercontent.com/216566/34265019-a864561a-e66c-11e7-8f58-1617461fce17.png)

**Comment by VasilyShelkov** [comment]
Here is my src/setupTests.ts file which does not display the error:

```js
// tslint:disable-next-line:no-any
(global as any).requestAnimationFrame = (cb: any) => {
  setTimeout(cb, 0)
}

import { configure } from 'enzyme'
import * as Adapter from 'enzyme-adapter-react-16'

configure({ adapter: new Adapter() })
```

it's key to put the `global.requestAnimationFrame` before the enzyme imports otherwise it doesn't work. Hope that helps

**Comment by cortopy** [comment]
According to this [issue](https://github.com/facebookincubator/create-react-app/issues/3199#issuecomment-332842582) the problem was resolved for the main scripts of create-react-app and there should be no need to mock requestAnimationFrame any more

Does anybody know if typescript scripts need updating or a different solution?

**Comment by cortopy** [comment]
Ah, yes, answering my own question, according to changelog react-scripts@1.0.13 was the last merge into this repo. This issue is solved in react-scripts@1.0.15. Hopefully 2.9.0 should fix this

**Comment by DorianGrey** [comment]
Should no longer be an issue in recent version.
